app/scripts/services/dropboxdatastore.coffee

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+###*
+ # @ngdoc service
+ # @name swarmApp.Dropboxdatastore
+ # @description
+ # # Dropboxdatastore
+ # Service in the swarmApp.
+###
+angular.module('swarmApp').factory 'Dropboxdatastore', ($log, util) -> class Dropboxdatastore
+  constructor: (@session) ->
+    $log.debug 'datastore constructed' + oath_token + "...."
+
+#angular.module('swarmApp') .service 'Dropboxdatastore', ->
+    # AngularJS will instantiate a singleton by calling "new" on this function
+
+  isAuthenticated: ->
+    return true;
+
+  oath_token = 'this is my oathtoekn';
```

Comment:
Don't think this syntax does what you want it to: http://jsfiddle.net/s6jbvvvf/

(I wasn't sure either, had to try it)